rdt/hyper_transformer.py

```diff
@@ -260,7 +260,7 @@ def reverse_transform(self, data):
                     # temporarily support both old and new style transformers
                     rename = dict(zip(columns, transformer.output_columns))
                     reversed_data = transformer.reverse_transform(data.rename(columns=rename))
-                    reversed_data = reversed_data[transformer.columns[0]].to_numpy()
+                    reversed_data = reversed_data[transformer.columns[0]]
```

Comment:
Is this change needed? Not a big issue, since this will be deprecated really soon, but needing this change may mean other things are wrong.

Reply:
Yeah, it crashes without it.